packages/validator-ajv8/src/processRawValidationErrors.ts

```diff
@@ -97,6 +102,7 @@ export function transformRJSFValidationErrors<
       params, // specific to ajv
       stack,
       schemaPath,
+      title: uiTitle,
```

Comment:
Thanks for the fix... can you add a unit test in `utils` to verify this fix?

Reply:
Sure! I'll try getting something done in the next few days.